samples/client/petstore/powershell_test/src/IO.Swagger/Model/New-Pet.ps1

```diff
@@ -0,0 +1,36 @@
+function New-Pet {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[Int64]]
+        ${ id },
+        [Parameter(Position = 0, ValueFromPipelineByPropertyName = $true)]
+        [System.Nullable[Category]]
```

Comment:
This, and others like this, should use full type name, e.g.: `[System.Nullable[IO.Swagger.Model.Category]]`